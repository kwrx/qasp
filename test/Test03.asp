%@exists
a | na.
b | nb.
%@forall
c | nc.
d | nd.
e | ne.
%@constraints
{c;d;e}.
:- c, not e.
:- d, not e.
:- not c, not d, e.