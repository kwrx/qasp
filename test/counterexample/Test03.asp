%@exists
a | na.
b | nb.
%@forall
c | nc.
d | nd.
e | ne.
%@constraints
:- c, not e.
:- d, not e.
:- not c, not d, e.