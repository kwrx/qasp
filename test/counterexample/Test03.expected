QASP 1.0

{b,a,ne,d,c}
{b,a,ne,nd,c}
{b,a,e,nd,nc}
{b,a,ne,d,nc}
{a,nb,ne,d,c}
{a,nb,ne,nd,c}
{a,nb,e,nd,nc}
{a,nb,ne,d,nc}
{nb,na,ne,d,c}
{nb,na,ne,nd,c}
{nb,na,e,nd,nc}
{nb,na,ne,d,nc}
{b,na,ne,d,c}
{b,na,ne,nd,c}
{b,na,e,nd,nc}
{b,na,ne,d,nc}
